### ISSUE TITLE
Please, add QA as a career path

### ISSUE THREAD

**Issue by elliot710** [anmedio/grades#39]

**Comment by NikolasMelui** [comment]
Maybe u want to help us?)

**Comment by NikolasMelui** [comment]
> _No description provided._

Maybe u want to help us?)

**Comment by elliot710** [comment]
> > _No description provided._
> 
> Maybe u want to help us?)

Yes, I could do that in the spare time. I'll come with a pull request

**Comment by NikolasMelui** [comment]
> > > _No description provided._
> > 
> > 
> > Maybe u want to help us?)
> 
> Yes, I could do that in the spare time. I'll come with a pull request

Great! Еhen we wait for the PR 👍
